notebooks/lst_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
<p>How is it defined exactly? Seems to get too good (~0.13) for a single telescope.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
<p>The definition according to pyirf is "Calculate the energy resolution as the half of the 68% containment percentile equivalent of the standard deviation.". The function can be found in pyirf/pyirf/benchmarks/energy_bias_resolution.py&nbsp;</p><p>It's true that its very good... these are MC data. Can be investigated. </p>